packages/core/scss/components/_radiobuttons.scss

```diff
@@ -8,13 +8,14 @@
  */
 
 @mixin radio-basic() {
-  $size: $large-space;
+  $size: 1.25rem;
 
   /* Base for label styling */
   [type='radio']:not(:checked),
   [type='radio']:checked {
     position: absolute;
-    opacity: 0;
+    display: none;
```

Comment:
@danielleroux Should we go back to opacity 0?

Reply:
Oh yes sure.

Reply:
@nuke-ellington can you please check.